Makefile.toml

```diff
@@ -0,0 +1,61 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
+
+[tasks.format]
+install_crate = "rustfmt"
+command = "cargo"
+args = ["fmt", "--", "--emit=files"]
+
+[tasks.clean]
+command = "cargo"
+args = ["clean"]
+
+[tasks.build]
+command = "cargo"
+args = ["build"]
+dependencies = ["clean"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
+dependencies = ["clean"]
+
+[tasks.deny]
+command = "cargo"
+args = ["deny"]
+
+[tasks.clippy]
+command = "cargo"
+args = ["clippy", "--", "-D warnings"]
+
+[tasks.cov]
+command = "cargo"
+args = [
+    "tarpaulin",
+    "--ignore-tests",
+    "-o",
+    "Lcov",
+    "--output-dir",
+    "target/tarpaulin",
+]
+
+[tasks.docs]
+command = "cargo"
+args = [
+    "doc",
+    "--target-dir",
+    "docs",
+    "--color",
+    "never",
+    "--no-deps",
+    "--open",
+    "--workspace",
+    "--exclude",
+    "'cosmwasm*'",
+    "--exclude",
+    "'cw*'",
+    "--release",
+]
+
+[tasks.all]
+dependencies = ["clean", "format", "build", "test", "deny", "clippy", "cov"]
```

Comment:
Set up to run in CI?

Reply:
Ok. Do we really need "clean" as the first command here? Seems unnecessary and will just slow the build time a lot?

Reply:
Removed it. And also removed all the tasks that are already run by default anyway. Pls read docs of tools you add before adding them!